docs/sphinx/cookbook/dynamic_pool_heuristics.rst

```diff
@@ -0,0 +1,43 @@
+.. _dynamic_pool_heuristics:
+
+=======================================
+Creating a Coalesce Heurisitic Function
+=======================================
+
+By default, a DynamicPool object will not coalesce blocks that it allocates
+for the pool.  As needed, the DynamicPool will continue to allocate blocks to
```

Comment:
This can be a link to the actual DynamicPool stuff:

```suggestion
for the pool.  As needed, the :class:`umpire::strategy::DynamicPool` will continue to allocate blocks to
```